packages/driver/src/cypress.js

```diff
@@ -31,6 +31,7 @@ const $SelectorPlayground = require('./cypress/selector_playground')
 const $utils = require('./cypress/utils')
 const $errUtils = require('./cypress/error_utils')
 const browserInfo = require('./cypress/browser')
+const debug = require('debug')('cypress-events:driver')
```

Comment:
why does this not follow the normal debug semantics?

```suggestion
const debug = require('debug')('cypress:driver:cypress')
```

later on...

```js
debug('action fired: %o', { eventName })
```